core/src/com/unciv/logic/civilization/managers/GreatPersonManager.kt

```diff
@@ -19,8 +19,10 @@ class GreatPersonManager : IsPartOfGameInfoSerialization {
     /** Base points, without speed modifier */
     var pointsForNextGreatPersonCounter = Counter<String>()  // Initial values assigned in getPointsRequiredForGreatPerson as needed
     var pointsForNextGreatGeneral = 200
+    var pointsForNextGreatGeneralCounter = Counter<String>() // Initial values assigned when needed
 
     var greatPersonPointsCounter = Counter<String>()
+    var greatGeneralPointsCounter = Counter<String>()
```

Comment:
Why do we need this as a separate counter? We can just use the regular greatPersonPointsCounter
The same unit shoudn't be able to be gained in both ways